nevergrad/benchmark/experiments.py

```diff
@@ -1268,16 +1269,25 @@ def image_quality(seed: tp.Optional[int] = None, cross_val: bool=False) -> tp.It
     optims: tp.List[tp.Any] = get_optimizers("structured_moo")
     if default_optims is not None:
         optims = default_optims
-    
+
     # We optimize func_blur or func_brisque and check performance on func_iqa.
     funcs: tp.List[ExperimentFunction] = [
-            imagesxp.Image(loss=imagesxp.imagelosses.Koncept512),
-            imagesxp.Image(loss=imagesxp.imagelosses.Blur),
-            imagesxp.Image(loss=imagesxp.imagelosses.Brisque),
-            ]
-    upper_bounds: tp.List[tp.Any] = [func(func.parametrization.sample().value) for func in funcs]
+        imagesxp.Image(loss=loss) for loss in (
+            imagesxp.imagelosses.Koncept512,
+            imagesxp.imagelosses.Blur,
+            imagesxp.imagelosses.Brisque,
+        )
+    ]
     # TODO: add the proxy info in the parametrization.
-    mofuncs = fbase.MultiExperiment.create_moo_crossvalidation_experiments(funcs, upper_bounds=upper_bounds, no_crossval=[1, 2], pareto_size=16) if cross_val else [fbase.MultiExperiment(funcs, upper_bounds=upper_bounds)]
+    if cross_val:
+        upper_bounds = [func(func.parametrization.value) for func in funcs]
+        mofuncs = [fbase.MultiExperiment(funcs, upper_bounds=upper_bounds)]  # type: ignore
+    else:
+        mofuncs = helpers.SpecialEvaluationFunction.create_crossvalidation_experiments(
```

Comment:
Maybe I'll understand later --- for now it's a mystery to me that there is no "upper_bounds" argument in 
SpecialEvaluationFunction.